src/AppBundle/Entity/Post.php

```diff
@@ -22,56 +22,74 @@
 class Post
 {
     /**
+     * @var int
```

Comment:
useless for constants. The IDE already reads the value anyway. A constant is not a variable